mentions/src/main/kotlin/io/spine/examples/pingh/mentions/GitHubClientServiceImpl.kt

```diff
@@ -59,17 +61,20 @@ public class GitHubClientServiceImpl(
     private val client = HttpClient(engine)
 
     /**
-     * Searches for user `Mentions` by the GitHub name of the user.
+     * Searches for user `Mentions` by the GitHub name of the user from the last update time.
```

Comment:
No, please rephrase. You cannot say this "from the last update time" and consider everyone gets what you mean.